### DISCUSSION TITLE
Eagle owl

### DISCUSSION THREAD

**Discussion by larsgunnarn** [Mattk70/Chirpity-Electron#113] [Q&A] [answered]
I an engaged in a Eurasian Eagle Owl project in Sweden. In my survey for breeding birds I use audio recording devices (AudioMoth) and about 200 files á 160 Mb need to be analysed. For this purpose I am using Audacity, where I analyse the frequency-time signal between 200 -800 Hz known to contain the call. However, Chirpity would be very helpful if I manage to make it work!
The attached file contains Eagle Owl calls, but Chirpity does not report the owl with the standard setting.
I should be very greatful with any hint on how to get Chiripityh to work correctly on this issue!
Best Regards,
Larsgunnar Nilsson
[L08_2024-02-29_1710.zip](https://github.com/Mattk70/Chirpity-Electron/files/14814530/L08_2024-02-29_1710.zip)

**Comment by Mattk70** [comment]
Hi @larsgunnarn, this is perhaps because you are using the default 'nocturnal birds' list. When I created this list for the Chirpity model, it was intended to filter out all species save for nocturnal **migrants**. I didn't consider the Eagle Owl to be a migrant species. This is perhaps debateable, and it is certainly a bird that vocalises at night. Anyway, if you use the Birds list, the Owl detections are shown: 

<img width="933" alt="Screenshot 2024-03-31 at 18 26 50" src="https://github.com/Mattk70/Chirpity-Electron/assets/61826357/55fc2591-d912-4d04-8111-56f6455c05aa">


It's possible you will get even better results if you do not first apply the cut filters. I'm not sure, but it's worth a try.

**Comment by larsgunnarn** [comment]
Hi Matt,
Thanks for the previous hint. I have attached another file with a recorded
Eagle-Owl. Chirpity does not recognize the Eagle-Owl although I am using
the Bird setting. The recorded Eagle-Owl sound is weak, but it can be heard
repeatedly.
Do you have any hint on how to make Chiripity recognise it?

Regards,
Larsgunnar
_________________


*Larsgunnar Nilsson+46-705-233539*


Den sön 31 mars 2024 kl 11:01 skrev Mattk70 ***@***.***>:

**Comment by Mattk70** [comment]
Hi @larsgunnarn, that file isn't attached. WIthout it, I can only suggest you try any or all of the following:

1. Use a custom list with just Eagle-Owl on it.
2. Drop the detection threshold down to 5%
3. Experiment with the audio filters. Before doing thie, enable the advanced option in settings to 'send filtered audio for analysis'.
4. Try both Chirpity and BirdNET models. One may be better than the other.

**Comment by larsgunnarn** [comment]
Thanks!
Yet another question:
The Eagle-Owl has several different sounds, for example
- Male call ("oohu",frequency about 350 Hz, )
- Female call ("oohu", frequency above 400 Hz)
- Femail call ("kvek-kvek")
In my recording,  XC361012, you can hear both the male's and the female's
"oohu"-sounds. In the recording  XC461798, the male's "oohu" call is
followed by the femails "kvek-kvek".

If I use a custom list in Chirpity with just the Eagle-owl named, would all
the different calls be recognised?

//Larsgunnar

_________________


*Larsgunnar Nilsson+46-705-233539*


Den mån 1 apr. 2024 kl 19:21 skrev Mattk70 ***@***.***>:

**Comment by Mattk70** [comment, marked as answer]
I ran the file through Chirpity here and all the calls seemed to be recognised in the first file. The model doesn't discern the sex of the caller - you won't get male detectinos or female detections, just Owl detections. The best results were with the raw audio - no filters, however, enabling context mode did pick out an additional call at the very beginning of the file:
<img width="600" alt="image" src="https://github.com/Mattk70/Chirpity-Electron/assets/61826357/835b302f-8646-4f25-aacb-55ee2c20b50b">

In the second file, the 'kvek-kvek' calls aren't recognised as Eagle-Owl. It associated that sound more with the Long-Eared Owl. I'm not sure whether LEO is an Owl you also get in this habitat, but you could add LEO to your custom list and check these detections for LEO vs. Eagle-Owl:

<img width="600" alt="image" src="https://github.com/Mattk70/Chirpity-Electron/assets/61826357/17ba6e2d-2c47-4858-bcd3-24e0439f1b95">

**Comment by Mattk70** [comment]
With the second file, the BirdNET model does appear to recognise the female calls quite well.
